server/data/prisonApiClient.ts

```diff
@@ -2,7 +2,7 @@ import RestClient from './restClient'
 import type { ApiConfig } from '../config'
 import config from '../config'
 import { prisonApiPaths } from '../paths'
-import type { Caseload, InmateDetail, OffenderSentenceAndOffences } from '@prison-api'
+import type { Caseload, InmateDetail, OffenderSentenceAndOffences, PageOffenceDto } from '@prison-api'
```

Comment:
`OffenceDto`? Not sure why this didn't error 🤔

Reply:
Was only using the paginated version `PageOffenceDto`, although will need to add `OffenceDto` now anyway when I change line 21 for the other comment.

Reply:
Oh yeah, that is in there 🤦🏼‍♀️